docs/source/changelog/experimental.md

```diff
@@ -0,0 +1,89 @@
+# Migrating from v0.19.0 to v0.20.0
+
+This constitutes a list of all the changes needed to migrate your code
+to work with the latest version of Manim
+
+## Manager
+If you ever used `Scene.render`, you must replace it with {class}`.Manager`:
+```py
+manager = Manager(SceneClass)
+manager.render()
+```
```

Comment:
I suggest adding an example of how the old code would have looked like.

I can't use the `suggestion` block, because the internal triple backticks end it too early, but I suggest something like the following (open this in edit mode if you want to copy the suggestion):

If you ever used `Scene.render`, you must rewrite your code to use {class}`.Manager`. For example:
```py
# Old code
scene = SceneClass()
scene.render()
```

should be changed to
```py
# New code
manager = Manager(SceneClass)
manager.render()
```